tahoe_sites/tests/test_backends.py

```diff
@@ -0,0 +1,126 @@
+"""
+Tests for Backend
+"""
+from unittest import mock
+
+import ddt
+from django.conf import settings
+from django.test import TestCase
+
+from tahoe_sites.api import create_tahoe_site
+from tahoe_sites.backends import DefaultSiteBackend, OrganizationMemberBackend
+from tahoe_sites.tests.fatories import UserFactory
+from tahoe_sites.tests.utils import create_organization_mapping
+
+
+class MixinTestBackendBase(TestCase):
```

Comment:
This test case works. I just think it's somewhat confusing. I recommend checking `pytest` fixtures for more convenient and streamlined fixtures: 

 - https://www.tutorialspoint.com/pytest/pytest_fixtures.htm
 - https://docs.pytest.org/en/6.2.x/fixture.html
 - https://docs.pytest.org/en/7.0.x/how-to/fixtures.html

No need to change anything yet, just to share few ideas with you.